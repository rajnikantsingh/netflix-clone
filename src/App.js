import Row from "./Row.js";
import "./App.css";
import requests from "./requests";

function App() {
	return (
		<div className="App">
			<h1>Rajni Netflix clone</h1>
			<Row
				title="Rajni Originals"
				fetchUrl={requests.fetchNetflixOriginals}
				isLargeRow
			/>
			<Row title="Rajni Trending Now" fetchUrl={requests.fetchTrending} />
			<Row title="Rajni Top Rated" fetchUrl={requests.fetchTopRated} />
			<Row title="Rajni Action Movies" fetchUrl={requests.fetchActionMovies} />
			<Row title="Rajni Comedy Movies" fetchUrl={requests.fetchComedyMovies} />
			<Row title="Rajni Horror Movies" fetchUrl={requests.fetchHorrorMovies} />
			<Row
				title="Rajni Romance Movies"
				fetchUrl={requests.fetchRomanceMovies}
			/>
			<Row title="Rajni Documentaries" fetchUrl={requests.fetchDocumentaries} />
		</div>
	);
}

export default App;
