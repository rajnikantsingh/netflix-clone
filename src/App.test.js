import React from "react";
import App from './App';
import renderer from 'react-test-renderer';

test('First snapshot Testing', () => {
  const component = renderer.create(<App />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
  
});
