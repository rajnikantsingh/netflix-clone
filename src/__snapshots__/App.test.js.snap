// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First snapshot Testing 1`] = `
<div
  className="App"
>
  <div
    className="nav"
  >
    <img
      alt="Netlfix Logo"
      className="nav_logo"
      src="https://upload.wikimedia.org/wikipedia/commons/6/67/NewNetflixLogo.png"
    />
    <img
      alt="Netlfix Avatar"
      className="nav_avatar"
      src="https://upload.wikimedia.org/wikipedia/commons/0/0b/Netflix-avatar.png"
    />
  </div>
  <header
    className="banner"
    style={
      Object {
        "backgroundImage": "url(
        \\"https://image.tmdb.org/t/p/original/undefined\\"
        )",
        "backgroundPosition": "center center",
        "backgroundSize": "cover",
      }
    }
  >
    <div
      className="banner_contents"
    >
      <h1
        className="banner_title"
      />
      <div
        className="banner__buttons"
      >
        <button
          className="banner_button"
        >
          Play
        </button>
        <button
          className="banner_button"
        >
          My List
        </button>
      </div>
      <h1
        className="banner_description"
      />
    </div>
    <div
      className="banner--fadeButtom"
    />
  </header>
  <div
    className="row"
  >
    <h2>
      Rajni Originals
    </h2>
    <div
      className="row_posters"
    />
    
  </div>
  <div
    className="row"
  >
    <h2>
      Rajni Trending Now
    </h2>
    <div
      className="row_posters"
    />
    
  </div>
  <div
    className="row"
  >
    <h2>
      Rajni Top Rated
    </h2>
    <div
      className="row_posters"
    />
    
  </div>
  <div
    className="row"
  >
    <h2>
      Rajni Action Movies
    </h2>
    <div
      className="row_posters"
    />
    
  </div>
  <div
    className="row"
  >
    <h2>
      Rajni Comedy Movies
    </h2>
    <div
      className="row_posters"
    />
    
  </div>
  <div
    className="row"
  >
    <h2>
      Rajni Horror Movies
    </h2>
    <div
      className="row_posters"
    />
    
  </div>
  <div
    className="row"
  >
    <h2>
      Rajni Romance Movies
    </h2>
    <div
      className="row_posters"
    />
    
  </div>
  <div
    className="row"
  >
    <h2>
      Rajni Documentaries
    </h2>
    <div
      className="row_posters"
    />
    
  </div>
</div>
`;
